src/components/cards/BorrowAmountCard.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import { div, mul } from 'biggystring'
+import * as React from 'react'
+import { View } from 'react-native'
+import FastImage from 'react-native-fast-image'
+
+import { DECIMAL_PRECISION } from '../../util/utils'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
+import { EdgeText } from '../themed/EdgeText.js'
+import { Card } from './Card'
+
+export const BorrowAmountCard = ({
+  dAppIcon,
+  fiatCode,
+  formattedAmount,
+  maxAmount,
+  title
+}: {
+  dAppIcon: string,
+  fiatCode: string,
+  formattedAmount: string,
+  maxAmount: string,
+  title: string
+}) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  return (
+    <Card>
+      <View style={styles.cardContainer}>
+        <View style={styles.leftContainer}>
+          <EdgeText style={{ fontFamily: theme.fontFaceMedium }}>{title}</EdgeText>
+          <View style={styles.valueContainer}>
+            <EdgeText style={styles.valueFont}>{formattedAmount}</EdgeText>
+            <EdgeText>{fiatCode}</EdgeText>
+          </View>
+          <View style={[styles.bar, { width: mul(div(formattedAmount, maxAmount, DECIMAL_PRECISION), '100') + '%' }]} />
+        </View>
+        <FastImage style={styles.icon} source={{ uri: dAppIcon }} />
```

Comment:
Don't pass inline Objects or Arrays as props, you need to memoize them using 'useMemo' or if it's a function then 'useCallback'